tools/Customization/DevHome.Customization/Views/DeveloperFileExplorerView.xaml.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using DevHome.Common.Extensions;
+using DevHome.Customization.ViewModels;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevHome.Customization.Views;
+
+public sealed partial class DeveloperFileExplorerView : UserControl
+{
+    public DeveloperFileExplorerViewModel ViewModel
+    {
+        get;
+    }
+
+    public DeveloperFileExplorerView()
```

Comment:
Suggest using `IHost` here to get the `ViewModel`

`Application.Current` throws an exception when ran in a unit test.

Reply:
Is there an example of a View doing this, they need a default constructor so I'm not sure how to pull that off offhand. I would suggest we file a follow-up task to do this across the project since most if not all existing ViewModels are acquired with this pattern.